src/main/java/edu/hm/hafner/analysis/parser/IdeaInspectionParser.java

```diff
@@ -61,6 +62,14 @@ else if ("ERROR".equals(severity)) {
         return priority;
     }
 
+    private String stripPathPrefix(final String file) {
+        String strippedFile = file;
+        if (file.startsWith(PATH_PREFIX)) {
```

Comment:
Is file guaranteed to be non `null`? Otherwise I would prefer using `StringUtils.startsWith`.

Or even simpler:
``` 
return StringUtils.removeStart(file, PATH_PREFIX);
```

Reply:
done